modules/nw-udn-cr.adoc

```diff
@@ -29,7 +29,7 @@ metadata:
 EOF
 ----
 
-. Create a request for either a `Layer2` or `Layer3` topology type user-defined network:
+.  Create a user-defined network for either a `Layer2` or `Layer3` topology type:
```

Comment:
Same here: We may create the resource and not get the network, but maybe that should be considered a CUDN creation failure.